src/hotspot/share/code/nmethod.hpp

```diff
@@ -58,9 +58,12 @@ class nmethod : public CompiledMethod {
   friend class NMethodSweeper;
   friend class CodeCache;  // scavengable oops
  private:
+  // nmethod's read-only data
+  address _immutable_data;
 
   // Shared fields for all nmethod's
   int       _entry_bci;        // != InvocationEntryBci if this nmethod is an on-stack replacement method
+  int       _immutable_data_size;
```

Comment:
It seems that field _immutable_data_size and _immutable_data are not initialized in constructor for native wrappers.

Reply:
Thanks for catch it.